docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+    | `SystemEvents.IncomingMsg.EventName`          | `SystemEvents.IncomingMsg.IdKey`                  | `msg_id`: Message ID of the received message (string of the unsigned `int` type value) | -                                         |           -                               |
+    | `SystemEvents.OutgoingMsg.EventName`          | `SystemEvents.OutgoingMsg.TypeKey`                | - `SystemEvents.OutgoingMsg.TypeValueSms`:  SMS-type message <br>-  `SystemEvents.OutgoingMsg.TypeValueMms`: MMS-type message | When an SMS or MMS message has been sent. |       -                                   |
+    | `SystemEvents.OutgoingMsg.EventName`          | `SystemEvents.OutgoingMsg.IdKey`                  | `msg_id`: Message ID of the received message (string of the unsigned `int` type value) | -                                         |           -                               |
+    
+- alarm-manager
+
+  - time
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.TimeChanged.EventName`          | N/A                     | N/A                                      | When  the system time setting has changed. | You  can get the current value with the `Applications.AlarmManager.GetCurrentTime` method. |
+
+- setting
+
+  - time
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.TimeZone.EventName`             | `SystemEvents.TimeZone.Key`               | The  value of this key is the time zone value of the time zone database, for  example, "Asia/Seoul", "America/New_York". For more  information, see the IANA Time Zone Database. | When  the time zone has changed.         | You  can get the current value with the `System.SystemSettings.LocaleTimeZone` property. |
+
+  - locale
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.HourFormat.EventName`           | `SystemEvents.HourFormat.Key`             | - `SystemEvents.HourFormat.Value12` <br>- `SystemEvents.HourFormat.Value24`                             | When  the `hour_format` has changed,  for example, by the user toggling the date and time settings for the 24-hour  clock (where **OFF** stands  for the 12-hour clock). | You  can get the current value with the `System.SystemSettings.LocaleTimeFormat24HourEnabled` property. |
+    | `SystemEvents.LanguageSet.EventName`          | `SystemEvents.LanguageSet.Key`           | The value of  this key is the full name of the locale, for example, `ko_KR.UTF8` for Korean and `en_US.UTF8` for American English. For more information, see the Linux  locale information. | When the `language_set` has changed.       | You can get the  current value with the `System.SystemSettings.LocaleLanguage` property. |
```

Comment:
Just a question. In this row the title is **SystemEvents.LanguageSet.EventName** while the data value description is about "locale", same to **SystemEvents.RegionFormat.EventName**'s description.
Would you please confirm that it is correct that **SystemEvents.LanguageSet.EventName** and **SystemEvents.RegionFormat.EventName**'s description are same?

Reply:
Yes, they are the same